sormas-e2e-tests/src/test/resources/features/sanity/web/Persons.feature

```diff
@@ -16,4 +16,20 @@ Feature: Edit Persons
     And While on Person edit page, I will edit all fields with new values
     And I edit all Person primary contact details and save
     Then I click on save button from Edit Person page
-    And I check that previous edited person is correctly displayed in Edit Person page
\ No newline at end of file
+    And I check that previous edited person is correctly displayed in Edit Person page
+
+@issue=SORDEV-8468
+  Scenario: Edit existent person and provoke errors
```

Comment:
let's be more specific with the place where we want to generate and check the errors, maybe similar tests will come up and we need to distinguish them from test name